skatepark/sass/blocks/_navigation.scss

```diff
@@ -2,4 +2,12 @@
 //preventing mobile menu items (and sub-menu items) from having a different size value
 .wp-block-navigation {
 	font-size: var(--wp--preset--font-size--small);
+
+	.wp-block-navigation-item {
+		&.current-menu-item > a {
+			text-decoration-line: underline;
+			text-decoration-thickness: 0.2em;
+			text-underline-offset: 0.35em;
```

Comment:
I wonder if we should use the [mixin](https://github.com/Automattic/themes/blob/trunk/skatepark/sass/base/_mixins.scss) that is used for this for the rest of the site. It won't look the same as the design, but it would be consistent with the rest of the underlines

Reply:
Ah yeah I like this idea, although it does look quite different from the design:

Custom for the design:

![image](https://user-images.githubusercontent.com/1645628/140042224-5a700363-2d56-44c2-9be8-9d58e2adabc9.png)

With this mixin:

![image](https://user-images.githubusercontent.com/1645628/140042305-a6f776a5-fbf5-4b3c-aa22-4128a1d4c27c.png)

Reply:
mmh yeah, don't make the changer then